src/components/standalone/dashboard/RealTimeTrafficCard.vue

```diff
@@ -0,0 +1,94 @@
+<!--
+  Copyright (C) 2023 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { ubusCall } from '@/lib/standalone/ubus'
+import { getAxiosErrorMessage, byteFormat1000 } from '@nethserver/vue-tailwind-lib'
+import { onMounted, onUnmounted, ref, type Ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+import NeCard from '@/components/NeCard.vue'
+import NeTable from '@/components/standalone/NeTable.vue'
+
+const { t } = useI18n()
+const REFRESH_INTERVAL = 10000
+const NUM_HOSTS = 5
+const topTalkersIntervalId: Ref<number> = ref(0)
+const topTalkers = ref([])
+
+const tableHeaders = [
+  {
+    label: t('standalone.dashboard.host'),
+    key: 'host'
+  },
+  {
+    label: t('standalone.dashboard.traffic'),
+    key: 'traffic'
+  }
+]
+
+let loading = ref({
+  getTopTalkers: false
+})
+
+let error = ref({
+  title: '',
+  description: ''
+})
+
+onMounted(() => {
+  getTopTalkers()
+
+  // periodically reload top talkers
+  topTalkersIntervalId.value = setInterval(getTopTalkers, REFRESH_INTERVAL)
+})
+
+onUnmounted(() => {
+  if (topTalkersIntervalId.value) {
+    clearInterval(topTalkersIntervalId.value)
+  }
+})
+
+function formatTraffic(value: number) {
+  value = Number(value.toFixed(2))
+  return byteFormat1000(value) + '\\s'
+}
+
+async function getTopTalkers() {
+  // show skeleton only the first time
+  if (!topTalkersIntervalId.value) {
+    loading.value.getTopTalkers = true
```

Comment:
If this is only for the first time, you could just set the value to `true` on line 32